starknet_py/net/http_client.py

```diff
@@ -37,8 +37,8 @@ async def request(
                 session=session,
                 address=address,
                 http_method=http_method,
-                params=params,
-                payload=payload,
+                params=params,  # pyright: ignore
+                payload=payload,  # pyright: ignore
```

Comment:
Why are these needed here?

Reply:
Becuse they don't use None as accepted type even though they default to it below.